src/ngraph/builder/quantization.cpp

```diff
@@ -121,23 +121,50 @@ namespace ngraph
                                            std::shared_ptr<Node> max_freezed_output,
                                            const bool with_relu)
         {
+            auto output_et = with_relu ? element::u8 : element::i8;
             auto requantization_scale = quantization_util::get_scale(min_input,
                                                                      max_input,
                                                                      min_filter,
                                                                      max_filter,
                                                                      min_freezed_output,
-                                                                     max_freezed_output);
+                                                                     max_freezed_output,
+                                                                     output_et);
 
-            return make_shared<op::QuantizedConvolutionBias>(input,
-                                                             filters,
-                                                             bias,
-                                                             window_movement_strides,
-                                                             window_dilation_strides,
-                                                             padding_below,
-                                                             padding_above,
-                                                             data_dilation_strides,
-                                                             requantization_scale,
-                                                             with_relu);
+            if (bias->get_element_type() != element::i32)
```

Comment:
First off, the QuantizedConvolutionBias experimental op should enforce that bias is of type element::i32.  The fact that those experimental ops omit type checking needs to change.  Second, I am not sure it should be the responsibility of this builder to meet those type requirements.  There isn't enough information (see rounding mode comment below) to make this conversion.  My feeling is that it should be the responsibility of the bridge to call these builders with properly formatted inputs.  If not the bridge, we likely need a graph pass to conincalize inputs.  What is the situation with TF DO graphs?  What is the type of the bias input we are converting from?